oas_docs/overlays/kibana.overlays.shared.yaml

```
@@ -61,6 +50,142 @@ actions:
+      x-displayName: 'Elastic Agent status'
+  - target: '$.tags[?(@.name=="Elastic Agents")]'
+    description: Change tag description and displayName
+    update:
+      description: >
+        Elastic Agents APIs enable you to manage Elastic Agents, including retrieving agent information, managing agent lifecycle, handling file uploads, and configuring agent setup.
+      x-displayName: 'Elastic Agents'
+  - target: '$.tags[?(@.name=="Fleet enrollment API keys")]'
+    description: Change tag description and displayName
+    update:
+      description: >
+        Fleet enrollment API keys APIs enable you to manage enrollment API keys for Fleet, including creating, retrieving, and revoking keys used for agent enrollment.
+      x-displayName: 'Fleet enrollment API keys'
+  - target: '$.tags[?(@.name=="Fleet internals")]'
+    description: Change tag description and displayName
+    update:
+      description: >
+        Fleet internals APIs enable you to manage Fleet internal operations, including checking permissions, monitoring Fleet Server health, managing settings, and initiating Fleet setup.
+      x-displayName: 'Fleet internals'
+  - target: '$.tags[?(@.name=="Fleet outputs")]'
+    description: Change tag description and displayName
+    update:
+      description: >
+        Fleet outputs APIs enable you to manage Fleet outputs, including creating, updating, and deleting output configurations, generating API keys, and monitoring output health.
+      x-displayName: 'Fleet outputs'
+  - target: '$.tags[?(@.name=="Fleet package policies")]'
+    description: Change tag description and displayName
+    update:
+      description: >
+        Fleet package policies APIs enable you to manage Fleet package policies, including creating, updating, and deleting policies, performing bulk operations, and managing policy upgrades.
+      x-displayName: 'Fleet package policies'
+  - target: '$.tags[?(@.name=="Fleet proxies")]'
+    description: Change tag description and displayName
+    update:
+      description: >
+        Fleet proxies APIs enable you to manage Fleet proxies, including creating, updating, and deleting proxy configurations for Fleet agent communication.
+      x-displayName: 'Fleet proxies'
+  - target: '$.tags[?(@.name=="Fleet Server hosts")]'
+    description: Change tag description and displayName
+    update:
+      description: >
+        Fleet Server hosts APIs enable you to manage Fleet Server hosts, including creating, updating, and deleting Fleet Server host configurations.
+      x-displayName: 'Fleet Server hosts'
+  - target: '$.tags[?(@.name=="Fleet uninstall tokens")]'
+    description: Change tag description and displayName
+    update:
+      description: >
+        Enables you to manage Fleet uninstall tokens, including retrieving metadata and decrypted tokens for agent uninstallation.
```

To match the structure for other Fleet APIs that have multiple endpoints:

```suggestion
        Fleet uninstall tokens APIs enable you to manage Fleet uninstall tokens, including retrieving metadata and decrypted tokens for agent uninstallation.
```